src/async_wrap.cc

```diff
@@ -504,18 +504,18 @@ void AsyncWrap::Initialize(Local<Object> target,
   FORCE_SET_TARGET_FIELD(                                                     \
       constants, #name, Integer::New(isolate, AsyncHooks::name));
```

Comment:
I think it would be better to drop the `;` here instead, because that way a `;` after the macro is still enforced (the `FORCE_SET_TARGET_FIELD` macro itself does the “right” thing here)